### ISSUE TITLE
Newly created user account lacks default group

### ISSUE THREAD

**Issue by 0rzech** [Antergos/Cnchi#629]
Just after installation, the user account created by Cnchi doesn't have it's default group. For instance, user _haxor_ should have default group with the same name _haxor_. As of now, after fresh installation, user _haxor_'s default group is _users_.

This introduces inconsistency, because Gnome's _Users_ app will create separate default group for each new user. It is also recommended on Arch's wiki to create user's default group, AFAIR.

**Comment by znmeb** [comment]
Yeah, this is a real pain point - the default for creating a user is to have the group be the same as the user. It's also the way just about every distro does it, and I think it's a security enhancment.
